doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst

```diff
@@ -0,0 +1,32 @@
+.. _bt_ll_acs_nrf53_changelog:
+
+Changelog
+#########
+
+.. contents::
+   :local:
+   :depth: 2
+
+Notable changes to this controller are documented in this file.
+
+Controller v3322
```

Comment:
Like @asbjornsabo previosly mentioned, the controller moved is not 3322, but 3330.
Is it correct to list v3322 here?

Reply:
Or rather: shouldn't we also list v3330 here, in addition to v3322? I would keep v3322, if it was used at some point and we have changelog changes documented for it.

Reply:
Done